db/migrate/20240109152343_create_users.rb

```diff
@@ -0,0 +1,11 @@
+class CreateUsers < ActiveRecord::Migration[7.1]
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.text :photo
+      t.string :bio
```

Comment:
- [ ] The `photo` should be of type `t.string`, while `bio` should be of type `t.text`

  ```
  t.string :photo
  t.text :bio
  ```

  Kindly ensure these changes are reflected in the schema file also.